tests/dualtor_mgmt/test_egress_drop_nvidia.py

```diff
@@ -0,0 +1,247 @@
+        upper_tor_host_name = upper_tor_mg_facts['inventory_hostname']
+        lower_tor_host_name = lower_tor_mg_facts['inventory_hostname']
+        upper_tor_loopback0_ip = upper_tor_mg_facts['minigraph_devices'][upper_tor_host_name]['lo_addr']
+        lower_tor_loopback0_ip = lower_tor_mg_facts['minigraph_devices'][lower_tor_host_name]['lo_addr']
+        lower_tor_uplink_ports = []
+        lower_tor_uplink_ptf_ports = []
+        for portchannel in lower_tor_mg_facts['minigraph_portchannels'].values():
+            for member in portchannel['members']:
+                lower_tor_uplink_ports.append(member)
+                ptf_indice = lower_tor_mg_facts['minigraph_ptf_indices'][member]
+                lower_tor_uplink_ptf_ports.append(ptf_indice)
+        lower_tor_vlan_ports = lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members']
+
+    with allure.step("Clear port counters"):
+        lower_tor_host.command("sonic-clear counters")
+
+    with allure.step("Start the traffic test"):
+        pkt = simple_tcp_packet(
+            eth_dst=dualtor_meta['vlan_mac'],
+            ip_src='1.1.1.1',
+            ip_dst=lower_tor_target_server_ip,
+            ip_ttl=64
+        )
+        # Send packets
+        ptfadapter.dataplane.flush()
+        testutils.send(ptfadapter, lower_tor_source_server_ptf_port, pkt, PKT_NUM)
+        # The packets can be forwarded by the upper tor to the target server
+        pkt.ttl -= 1
+        expected_packet = get_masked_packet(pkt)
+        # Check the expected packets are received by the server ptf port
+        testutils.verify_packet(ptfadapter, expected_packet, lower_tor_target_server_ptf_port)
+
+        # There should not be any tunnel traffic sent by the lower tor uplink ports
+        tunnel_pkt = simple_ipv4ip_packet(
+            eth_dst=upper_tor_host.facts['router_mac'],
+            eth_src=lower_tor_host.facts['router_mac'],
+            ip_src=lower_tor_loopback0_ip,
+            ip_dst=upper_tor_loopback0_ip,
+            inner_frame=pkt[IP])  # noqa F821
+        not_expected_tunnel_pkt = get_masked_packet(tunnel_pkt, is_tunnel_packet=True)
+        # Send packets again
+        ptfadapter.dataplane.flush()
+        testutils.send(ptfadapter, lower_tor_source_server_ptf_port, pkt, PKT_NUM)
+        # Check the tunnel packets are not received by any uplink ptf ports of the lower tor
+        testutils.verify_no_packet_any(ptfadapter, not_expected_tunnel_pkt, lower_tor_uplink_ptf_ports)
+
+    with allure.step("Check the port counters to make sure the drop on lower tor happened"):
+        # There are in total 2 * PKT_NUM packets sent, the rx counter value of the source port on the lower tors
+        # should be slightly lager than 2 * PKT_NUM.
+        # And all the port tx counters on the lower tor should be slightly lager than zero.
+        rx_counter_range = [PKT_NUM * 2, PKT_NUM * 2 * 1.05]
+        tx_counter_range = [0, PKT_NUM * 0.05]
+
+        def _check_counters():
```

Comment:
The counter is not cleared before reading. That may increase flakiness

Reply:
The counter is cleared in [Line170](https://github.com/sonic-net/sonic-mgmt/pull/9619/files/b8b5b2152e6d8028eaf9c9ed01bbace15ad4d2e8#diff-24f8b7ba2d894c20f16ef5408f9264cc7cec8b1b43c756ec2b65d993c0356ec7R169-R170). Only the counters on the lower(standby) tor are checked in this test case.